_board/circuitplayground_bluefruit.md

```diff
@@ -50,7 +50,8 @@ on the go!
 
 ## Tutorials
 
-* [Circuit Playground Overview](https://learn.adafruit.com/adafruit-circuit-playground-bluefruit)
+* [Circuit Playground Bluefruit Overview](https://learn.adafruit.com/adafruit-circuit-playground-bluefruit)
+* [Projects and Guides](https://learn.adafruit.com/search?q=Circuit%20Playground%20Bluefruit)
```

Comment:
Instead of doing a search, there are pages that show every guide that is associated with a product: https://learn.adafruit.com/products/4333/guides

This is based on what products are in the sidebar and I'd expect it to give better results.

Reply:
Neat! I had no idea these existed. I will change the search links later tonight.

Reply:
Yup! The only way I know how to get them is from the product page. Having them here will make them better known. Thanks for updating them!